denops/glance/main.ts

```diff
@@ -20,37 +35,63 @@ export async function main(denops: Denops) {
   }
 
   async function update() {
+    renderer = await ensureRenderer();
+    server = await ensureServer();
     const lines = await fn.getline(denops, 1, "$");
     const content = lines.join("\n");
     const document = await renderer.render(content);
     const pos = await fn.getpos(denops, ".");
     server.send("update", { document, line: pos[1] });
   }
-  const port = (await g.get(denops, "glance#server_port", 8765))!;
-  const plugins = (await g.get(denops, "glance#markdown_plugins", []))!;
-  const html = (await g.get(denops, "glance#markdown_html", false))!;
-  const breaks = (await g.get(denops, "glance#markdown_breaks", false))!;
-  const linkify = (await g.get(denops, "glance#markdown_linkify", false))!;
-  const defaultStylesheet = `#root {margin: 50px auto; width: min(700px, 90%);}`;
-  const stylesheet = (await g.get(denops, "glance#stylesheet", defaultStylesheet))!;
-  const defaultConfigPath = new URL("./config.ts", import.meta.url).toString();
-  const configPath = (await g.get(denops, "glance#config", defaultConfigPath))!;
-  const { createMarkdownRenderer } = await import(configPath);
-  const renderer = new MarkdownRenderer();
-  await renderer.initialize({ html, breaks, linkify, plugins, createMarkdownRenderer });
-  const server = new Server({ onOpen: update, readFile, stylesheet });
+
+  async function ensureOptions() {
+    if (options) return options;
+    const defaultStylesheet = `#root {margin: 50px auto; width: min(700px, 90%);}`;
```

Comment:
There is no string interpolation. The template literal is unneeded.

```suggestion
    const defaultStylesheet = '#root {margin: 50px auto; width: min(700px, 90%);}';
```

Reply:
Sure. Fixing.